SU-SWS/sulgryphon-nextjs
# SUL23-60 Style the header and footer components

# READY FOR REVIEW

# Summary
- Styles and interaction for the main menu
- Styles for the secondary navigation on the left sidebar.

# Steps to Test

1. view the preview link from netlify
2. interact with the main menu in mobile and on desktop
3. verify you can access everything and it function as you would expect it to.



<a href="https://gitpod.io/#https://github.com/SU-SWS/sulgryphon-nextjs/pull/30"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
